subsys/shell/modules/Kconfig

```diff
@@ -13,3 +13,8 @@ config DEVICE_SHELL
 	bool "Enable device shell"
 	help
 	  This shell provides access to basic device data.
+
+config DATE_SHELL
+	bool "Enable date shell"
```

Comment:
There definitely should be `depends on POSIX_CLOCK` here.

Reply:
Thanks, done!